platform/ios/src/MGLAnnotationView.h

```diff
@@ -171,6 +171,10 @@ MGL_EXPORT
 
  The default value of this property is `YES`. Set this property to `NO` if the
  view’s legibility is important.
+
+ @note Scaling many on-screen view annotations can contribute to poor map
```

Comment:
Nit: “view-backed annotations” or “annotation views”, but not “view annotations”. (All annotations go inside a view, the map view.)